dashboard/apps/prototype/management/commands/helpers.py

```diff
@@ -73,3 +81,71 @@ def print_task(task, start_date, end_date, padding='  '):
         else:
             logger.info('%s  %s', padding, line)
     return time_spent, money_spent
+
```

Comment:
Should most of this file be put in to custom Managers or Querysets for each model?